scripts/github-status/github-status.rb

```diff
@@ -117,6 +117,10 @@ def show_forcerebuild_pull_requests
     options[:pr] = pr
   end
 
+  opts.on('-r', '--repository REPO', 'Github Org Repository') do |repo|
```

Comment:
Good point, I agree `ORG/REPO` is clearer and less typing than an extra `--organization` option.